src/cart/crt_hg_proc.c

```diff
@@ -595,7 +595,7 @@ crt_proc_in_common(crt_proc_t proc, crt_rpc_input_t *data)
 		D_GOTO(out, rc);
 	}
 out:
-	return rc;
+	return crt_der_2_hgret(rc);
```

Comment:
Line 519 does direct return so needs to be modified as well

Reply:
Sigh, sorry. Fixed.